examples/tfengine/modules/team.hcl

```diff
@@ -103,13 +103,18 @@ template "project_networks" {
         ]
         cloud_sql_private_service_access = {} # Enable SQL private service access.
       }]
+      service_accounts = [{
+        account_id   = "bastion-accessor"
+        description  = "Placeholder service account to use as members who can access the bastion host."
+        display_name = "Bastion Accessor Service Account"
+      }]
       bastion_hosts = [{
         name           = "bastion-vm"
         network        = "$${module.network.network.network.self_link}"
         subnet         = "$${module.network.subnets[\"{{.default_location}}/bastion-subnet\"].self_link}"
         image_family   = "ubuntu-2004-lts"
         image_project  = "ubuntu-os-cloud"
-        members        = ["group:{{.prefix}}-bastion-accessors@{{.domain}}"]
+        members        = ["serviceAccount:$${google_service_account.bastion_accessor.email}"] # Placeholder for testing.
```

Comment:
How come this needs to be a serviceAccount and not a group?

Reply:
because otherwise we need to supply an existing group in the test. Using a service account created on the fly to avoid that. It's also how CFT's tests do it for IAM members.